packages/s3/manifest.ttl

```diff
@@ -16,3 +16,10 @@
   code:implementedBy [ a code:EcmaScriptModule;
     code:link <node:barnard59-s3/cmd/putObject.js#default>
   ].
+
+<getStreamObject> a p:Operation;
+  rdfs:label "Get S3 object as stream";
+  rdfs:comment "Get an object from a S3 bucket as a Readable stream.";
+  code:implementedBy [ a code:EcmaScriptModule;
+    code:link <node:barnard59-s3/cmd/getStreamObject.js#default>
+  ].
```

Comment:
What do you think about calling this `s3:getObject.stream`?

Reply:
With the dot in the middle, it looks a bit strange?
But I can call it `getObjectStream` if you think it's better?

Reply:
Trying find a pattern, some step use slash. Like `<json/parse>` and `<json/stringify>`

The upside of dot is that it does not need to be escaped in prefixed form

Reply:
If `getObjectStream` is not OK, I suggest going with `getObject/stream`, so that it has the same pattern as the other operations.
Importing it as a dedicated prefix is not an issue.

I just did a commit to go in that direction.

But now the pattern of that operation name doesn't reflect the name of the file…

Reply:
> But now the pattern of that operation name doesn't reflect the name of the file…

I think we can accept that :)